package-lock.json

```diff
@@ -661,9 +661,9 @@
       }
     },
     "@openzeppelin/contracts": {
-      "version": "4.5.0",
```

Comment:
Esse arquivo não precisa ser alteado! Podes dar um rollback nele pfv?

Quando for fazer o commit pode descartar alterações que não devem ser commitas